### PR TITLE
docs(3.10.11): add breaking change regarding platform policies

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

https://github.com/gravitee-io/issues/issues/7138
https://github.com/gravitee-io/issues/issues/7143

### CODE REVIEW

File: upgrades/3.x/3.10.11/README.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+The goal of policies at the platform level is to define them once and execute them for any APIs instead of having to duplicate the same configuration for each of the APIs.
+
+Regarding the execution order of the policies, the behavior before 3.10.11 was:
+
+- Platform request policies were executed before API request policies
+- Platform response policies were executed before API response policies
+
+To ensure the platform policies are always the first and last policies to be applied, and thus enforcing some behavior at the platform level, the execution order has been changed.
+As of `3.10.11` the following rules apply:
```
Comment (ytvnr) [inline upgrades/3.x/3.10.11/README.adoc:13]:
```suggestion
To ensure the platform policies are always the first and last policies to be applied, and thus enforcing some behavior at the platform level, the execution order has been changed.

As of `3.10.11` the following rules apply:
```

**Review by ytvnr — APPROVED** [review]
I just would have passed a line

**Review by marcambier — APPROVED** [review]

**Comment by phiz71** [comment]
⚠️ Wait the 3.10.11 release before merging it ⚠️

**Comment by phiz71** [comment]
Finally, I created a new branch (like on gravitee-docs) to merge all 'upgrade guide' PR in it.

**Pull request merged**